api/v1alpha3/virtualmachine_types.go

```diff
@@ -197,6 +197,19 @@ const (
 	// ManagerID on a VirtualMachine contains the UUID of the
 	// VMware vCenter (VC) that is managing this virtual machine.
 	ManagerID = GroupName + "/manager-id"
+
+	// +kubebuilder:validation:Enum=ImportVM;RegisterVM
```

Comment:
Please do the following to make it consistent with our other enums.

```go
+kubebuilder:validation:Enum=ImportVM;RegisterVM
type SourceAPI string

const (
    SourceAPIImportVM SourceAPI = "ImportVM"

    SourceAPIRegisterVM SourceAPI = "RegisterVM"
)
```

Then you can use the above instead of defining strings in the constants package.